types/foundry-vtt-api/types/apps/hud/players.d.ts

```diff
@@ -0,0 +1,27 @@
+// @TODO:
+
+/**
+ * The active Player List application
+ */
+declare class PlayerList extends Application {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/apps/hud/players.d.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.